src/modules/aws_ec2/entity/key_pair.ts

```diff
@@ -0,0 +1,44 @@
+import { Column, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn, Unique } from 'typeorm';
+
+import { KeyType } from '@aws-sdk/client-ec2';
+
+import { cloudId } from '../../../services/cloud-id';
+import { AwsRegions } from '../../aws_account/entity';
+
+@Entity()
+@Unique('uq_keypair_id_region', ['id', 'region'])
+@Unique('uq_keypair_name_region', ['name', 'region'])
+export class KeyPair {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  @cloudId
+  name: string;
+
+  @Column({ nullable: true })
+  keyPairId?: string;
+
+  @Column({
+    nullable: true,
+    type: 'enum',
+    enum: KeyType,
+  })
+  type: KeyType;
```

Comment:
nit: should we use the datatype from `aws-sdk`?

Reply:
I have followed this approach on other entities, to avoid redefining the types that were already there and could be used from AWS. Do you think it would give some problems?